presto-main/src/main/java/com/facebook/presto/operator/scalar/annotations/ParametricScalarImplementation.java

```diff
@@ -82,7 +88,7 @@
 import static java.lang.reflect.Modifier.isStatic;
 import static java.util.Objects.requireNonNull;
 
-public class ScalarImplementation
+public class ParametricScalarImplementation
```

Comment:
Why `Move @BlockPosition and @BlockIndex to SPI` commit moves other classes around as well?

Reply:
@findepi : Separated into a different commit now.